frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/useBuilderFields.ts

```diff
@@ -64,5 +66,7 @@ export const useBuilderFields = () => {
     return existingFields
   }, [formData, stateData])
 
-  return { builderFields }
+  return {
+    builderFields: builderFields?.map((field) => augmentWithMyInfo(field)),
```

Comment:
should we map this inside the memoized builderfields constant instead of here?

Reply:
good catch, changed accordingly